rfc/text/advanced/ap_10_rpc_progressive_calls.md

```diff
@@ -259,3 +259,15 @@ to be matched pairs.
 
 Because they are part of the same call operation, the request ID is the same in all `CALL`,
 `INVOCATION`, `YIELD`, and `ERROR` messages in the above exchange.
+
+
+**Stickiness of Progressive Call Options**
+
+Except for `progress`, items in the `Options` dictionary of the initiating progressive `CALL` shall be effective for
+the entirety of the progressive call request. Except for `progress`, all options in subsequent progressive calls
+(within the same overall request) shall be ignored by the *Dealer*. Except for `progress`, items in the `Details`
+dictionary of the corresponding `INVOCATION` message shall be based on the initiating progressive `CALL` only.
```

Comment:
By the way, it might be worth noting that any options (besides `progress`) can be omitted altogether in subsequent calls.

Reply:
Agreed. I'll add that note.

Reply:
Not having to repeat (and not being able to change) options is more in line with the idea of a media stream where you set things up initially, and just keep sending more data thereafter.

Reply:
> Not having to repeat (and not being able to change) options is more in line with the idea of a media stream where you set things up initially, and just keep uploading more data thereafter.

I think that comment (or similar) deserves to be added to the spec as rationale for _why_ progressive call options are sticky.

Reply:
yeah, agree!